src/PerformanceTests/Utils/QueueUtils.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+using System.Messaging;
+
+static class MsmqUtils
```

Comment:
I'll keep it in because we need logic to cleanup / remove the queues after a run.